schemas/context/enduserids.example.1.json

```diff
@@ -1,18 +1,16 @@
 {
-  "xdm:realm": {
-    "@id": "https://data.adobe.io/experiencecloud/audiencemanager",
-    "xdm:name": "Adobe Audience Manager"
+  "https://ns.adobe.com/experience/mcId": {
```

Comment:
We should use all-lowercase URIs for consistency.

Reply:
Sounds good. As I mentioned in another comment these URI's within the data platform have to be converted into a nested structure where the "mcId" becomes a property of the parent "experience" object (ns.adobe.com.experience.mcId). We can just add logic to our translation of the schemas to convert that final field name into camelCase.

Reply:
@cmathis @trieloff  how to convert to camelCase if the text are all lowercase without '-' used??

Reply:
@jwen-adobe - Good point Joe. Didn't think that one through. If the final value in the URI is lowercase then we have no way to get back to camelCase. A decision is needed to know how critical it is if all final storage field names need to be camelCase.